### PR TITLE
add test for 3783 issue

### PR DESCRIPTION
**Pull request by iCollin**
ATF Test Scripts to check https://github.com/smartdevicelink/sdl_core/issues/3783

This PR is **ready** for review.

### Summary
Add test for issue 3783
In my testing this fails about 1-2% against the 7.0 release

### CLA
- [x] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

File: test_sets/Defects/Defects_release_8_0.txt [hunk 1]
```diff
@@ -35,3 +35,4 @@
 ./test_scripts/Defects/8_0/3788_2_allow_RC_permission_for_App_after_PTU_with_REMOTE_CONTROL_appHMIType.lua
 ./test_scripts/Defects/8_0/3788_3_revoke_RC_permission_for_App_after_PTU_with_MEDIA_appHMIType.lua
 ./test_scripts/Defects/8_0/3797_3798_HMI_sends_SetAppProperties_and_tries_to_ActivateApp.lua
+./test_scripts/Defects/8_0/3783_crash_during_vr_langchange.lua
```
Comment (jacobkeeler) [inline test_sets/Defects/Defects_release_8_0.txt:38]: Probably should move this to `8_1` now

Comment (dboltovskyi) [reply test_sets/Defects/Defects_release_8_0.txt:38]: @jacobkeeler Please notice related issue https://github.com/smartdevicelink/sdl_core/issues/3783 had been fixed in scope of `8.0` that's why script is in corresponding test set

File: test_scripts/Defects/8_0/3783_crash_during_vr_langchange.lua [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+end
+
+local function registerAllApps()
+  for i = 1,5 do
+    common.getMobileSession(i, 1):SendRPC("RegisterAppInterface", common.app.getParams(i))
+  end
+  utils.wait(250)
+  for j = 1,5 do
+    common.getMobileSession(j):SendRPC("UnregisterAppInterface", {})
+  end
```
Comment (jacobkeeler) [inline test_scripts/Defects/8_0/3783_crash_during_vr_langchange.lua:59]: For this issue specifically, I don't think we are supposed to manually unregister the apps, they should be unregistered automatically due to the language change

**Review by jacobkeeler — CHANGES_REQUESTED** [review]

File: test_scripts/Defects/8_0/3783_crash_during_vr_langchange.lua (hunk 2 above)
Comment (iCollin) [reply test_scripts/Defects/8_0/3783_crash_during_vr_langchange.lua]: I have removed extraneous steps from the test in https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/2603/commits/3acb6fd631731f53df8aa6ece82d4744f42d30ae

**Review by jacobkeeler — APPROVED** [review]

**Pull request merged**